tests/p2p/test_service.py

```diff
@@ -27,8 +27,10 @@ class WaitService(BaseService):
     service = ParentService()
     asyncio.ensure_future(service.run())
     await asyncio.sleep(0.01)
+    assert service.daemon.is_operational
```

Comment:
Maybe it's just the diff but the asserts and awaits are all running together visually for me, maybe add some line breaks if you get back in here for other things?